src/components/status/StatusPairing.js

```diff
@@ -16,7 +16,25 @@ import {
 import { useDialogContext } from "../utils/DialogComponent";
 import StatusTable from "./StatusTable";
 
-function createMetadataObjectFromTNPairing(TNPairingResponse) {
+async function getWorkflowIdFromLibraryId(library_id){
+
+  const APIConfig = {
+    queryStringParameters: {
+      library_id: library_id,
+    },
+  };
+
+  const library_run_response = await API.get(
+    "DataPortalApi",
+    "/libraryrun/",
+    APIConfig
+  );
+  
+  const library_run = library_run_response.results[0]
+  return library_run.workflows
```

Comment:
You are not including the sequence run ID?
This will query for LibraryRuns that match that library ID right? That will still match records from different sequence runs, no?

So just taking `library_run_response.results[0]` may not be correct?

Reply:
This is for the TR pairing when there is a second table when you click the link button.

I used the pairing endpoint and does not return the instrument_run_id except the `rgid` which contain the intrument_run_id. Would you think splitting and parsing the instrument_run_id from the `rgid` is good enough ?